### ISSUE TITLE
GCSFuse can no longer be installed from the official repo (502  Bad Gateway)

### ISSUE THREAD

**Issue by pshved** [GoogleCloudPlatform/gcsfuse#1428]
**Describe the issue**

The official installation instructions no longer work because the packages for gcsfuse are broken in the official repo packages.cloud.google.com; Dockerfiles give an error when trying to install it.  The rest of the repo is functional; only GCSFuse packages seem to be affected.

Feel free to close as a duplicate of #1424 which describes the same issue under a different title

Minimal Dockerfile (taken from the official docs at https://cloud.google.com/run/docs/tutorials/network-filesystems-fuse#cloudrun_fs_dockerfile-python)

```
FROM ubuntu:focal

RUN set -e; \
    apt-get update -y && apt-get install -y \
    curl \
    gnupg2 \
    lsb-release; \
    gcsFuseRepo=gcsfuse-`lsb_release -c -s`; \
    echo "deb http://packages.cloud.google.com/apt $gcsFuseRepo main" | \
    tee /etc/apt/sources.list.d/gcsfuse.list; \
    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
    apt-key add -; \
    apt-get update; \
    apt-get install -y gcsfuse
```

This prints this error and fails to install the package

```
#0 17.38 Err:2 http://packages.cloud.google.com/apt gcsfuse-focal/main amd64 gcsfuse amd64 1.2.0
#0 17.38   502  Bad Gateway [IP: 142.251.46.238 80]
#0 17.47 Get:3 http://archive.ubuntu.com/ubuntu focal/main amd64 fuse amd64 2.9.9-3 [26.0 kB]
#0 17.49 E: Failed to fetch http://packages.cloud.google.com/apt/pool/gcsfuse-focal/gcsfuse_1.2.0_amd64_0e91d38c68c91349633d37c17e1586e6.deb  502  Bad Gateway [IP: 142.251.46.238 80]
#0 17.49 E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
#0 17.49 Fetched 108 kB in 0s (251 kB/s)
------
Dockerfile:3
--------------------
   2 |     
   3 | >>> RUN set -e; \
   4 | >>>     apt-get update -y && apt-get install -y \
   5 | >>>     curl \
   6 | >>>     gnupg2 \
   7 | >>>     lsb-release; \
   8 | >>>     gcsFuseRepo=gcsfuse-`lsb_release -c -s`; \
   9 | >>>     echo "deb http://packages.cloud.google.com/apt $gcsFuseRepo main" | \
  10 | >>>     tee /etc/apt/sources.list.d/gcsfuse.list; \
  11 | >>>     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
  12 | >>>     apt-key add -; \
  13 | >>>     apt-get update; \
  14 | >>>     apt-get install -y gcsfuse
  15 |    
```

None of the directories are available (My browser shows Error 501 and a sad google robot): 
* https://packages.cloud.google.com/apt/dists/gcsfuse-focal
* https://packages.cloud.google.com/apt/dists/gcsfuse-xenial
* etc

Other packages are working, e.g. [this link](https://packages.cloud.google.com/apt/dists/gce-google-compute-engine-stable) and packages reachable from [the index](https://packages.cloud.google.com/apt/dists/)

**Comment by mosmos6** [comment]
Errored the same here too.

**Comment by gargnitingoogle** [comment]
The error in https://github.com/GoogleCloudPlatform/gcsfuse/issues/1428#issue-1933502519 does look exactly the same as in https://github.com/GoogleCloudPlatform/gcsfuse/issues/1424#issue-1927877132 .

@pshved does the solution in https://github.com/GoogleCloudPlatform/gcsfuse/issues/1424#issuecomment-1749161971 unblock you?

**Comment by pshved** [comment]
Yes that workaround fixes it for us. A few of our developers will run into the same problem until they know they should merge the workaround.  Not resolving this will cost hundreds of engineers hours of productivity spent on indetifying, deploying, and distributing workaround.

**Comment by gargnitingoogle** [comment]
> Yes that workaround fixes it for us. A few of our developers will run into the same problem until they know they should merge the workaround. Not resolving this will cost hundreds of engineers hours of productivity spent on indetifying, deploying, and distributing the workaround.

@pshved  Thanks. I'll mark this as a duplicate of https://github.com/GoogleCloudPlatform/gcsfuse/issues/1424 . Please follow/subscribe to that for updates on this issue.
